www/student.js

```diff
@@ -21,6 +21,7 @@ function checkOutput(output) {
     label.textContent = 'Correct   ✔'
     label.classList.add('labelCorrect')
     label.classList.remove('labelIncorrect')
+    clearInterval(timer)
```

Comment:
The value passed to `clearInterval` should be the id value returned from `setInterval`

MDN has an example that may be informative: https://developer.mozilla.org/en-US/docs/Web/API/Window/setInterval#example_2_alternating_two_colors